### DISCUSSION TITLE
Problems with IdeaVIM after v2.5.1

### DISCUSSION THREAD

**Discussion by jlandahl** [JetBrains/ideavim#757] [Q&A]
I know that IdeaVIM v2.6.x was reported as broken, and since then I've been sticking with v2.5.1. I think I've tried every new release since v2.5.1 (just tried v2.7.3) and each one of them has the same problem: when in insert/add mode, I have to press Esc 2-3 times before it gets me back to command mode. Reverting back to v2.5.1 fixes the problem.

Is anyone else running into this?

**Comment by AlexPl292** [comment]
Could you please attach log (https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files)? You can also create a ticket on [YouTrack](https://youtrack.jetbrains.com/issues/VIM) and attach the log with limited visibility.

**Comment by jlandahl** [comment]
It looks like it's related to using Emacs as the underlying keymap setting (I'm a longtime Emacs user, and in Doom Emacs I sometimes fall back on Emacs keystrokes for some things, like Ctrl+L to center the current line on the screen). IdeaVim v2.7.5 is now telling me that Esc is misconfigured, but when I let it fix the problem it completely breaks the Esc key for me (i.e. it now never gets me out of edit mode).

Changing the keymap back to Gnome seems to fix the problem.
